src/app/events/page.tsx

```diff
@@ -93,6 +96,22 @@ export default function Events() {
       <PageSection kicker="Events" title="All Events">
         <EventsClient events={events} />
       </PageSection>
+
+      <PageSection
+        kicker="Get Involved"
+        title="Get in Touch With the Events Team"
+      >
+        <CardLayout>
+          {getInvolvedData.map(({ title, description, cta }) => (
+            <Card
+              key={title}
+              title={title}
+              description={description}
+              cta={{ ...cta }}
```

Comment:
why spread in an object? it's already an object?

Reply:
I guess some refactor gone wrong. 🙏🏼